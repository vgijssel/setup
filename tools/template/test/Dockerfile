FROM {{ base_image.value }}
ENV VERSION={{ version.value }}

ADD {{ output_file.out_location }} /output