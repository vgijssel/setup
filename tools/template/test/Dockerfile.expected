FROM some:image
ENV VERSION=10.0.0

ADD plz-out/gen/tools/template/test/output_file /output